src/api/libcellml/units.h

```diff
@@ -457,7 +465,7 @@ class LIBCELLML_EXPORT Units: public NamedEntity, public ImportedEntity
      * @return @c true if the two @c Units are dimensionally equivalent, @c false
      * otherwise.
      */
-    static bool dimensionallyEquivalent(const UnitsPtr &units1, const UnitsPtr &units2);
+    static bool equivalent(const UnitsPtr &units1, const UnitsPtr &units2);
```

Comment:
If we are to change the name of the methods (i.e. `equivalent()` -> `compatible()` and `dimensionallyEquivalent()` -> `equivalent()`) then we also ought to update their documentation.

Reply:
Done :)